odoo/addons/base/ir/ir_qweb/fields.py

```diff
@@ -374,6 +374,8 @@ def value_to_html(self, value, options):
         if minutes == 60:
             minutes = 0
             hours += 1
+        if hours == 0 and value < 0:
```

Comment:
How about (instead of the if):
```
sign_str = '-' if sign == -1 else ''
return '%s%02d:%02d' % (sign, hours, minutes)
```

Reply:
We had another aproach for our own fix:

```
    @api.model
    def value_to_html(self, value, options):
        hours, minutes = divmod(abs(value) * 60, 60)
        if value < 0:
            return '-%02d:%02d' % (hours, minutes)
        return '%02d:%02d' % (hours, minutes)
```
Very simple, not very beautiful but worked for a long time...